Makefile

```diff
@@ -19,6 +19,7 @@ test: builddir npm
 	LANG=en_US.utf-8 LC_ALL=en_US.utf-8 cd sippy-ng; CI=true npm test -- --coverage
 
 lint: builddir npm
+	cd sippy-ng; npm audit --production
```

Comment:
Can we move the comment on line 29 and 30 up here too?

Reply:
Do we also need to `cd ..`? Or perhaps move this below go-lint.sh.

Reply:
Yup I was surprised too.